examples/NewPassManager/input/compile.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+
+SOURCE_FILE=analysis_test.c
+
+echo "#include <stdio.h> " >> $SOURCE_FILE
```

Comment:
haha huge yikes to an `echo` per line.  I'm probably missing something... why not just make the `analysis_test.c` be a separate file instead of generating it in this sh script?

Reply:
did it that way so I could run it from wherever I wanted, and not have to worry about a path.